src/device-state/current-state.ts

```diff
@@ -244,12 +228,19 @@ function reportCurrentState(attempts = 0) {
 						maxDelay,
 						// Start the polling at the value given by the API if any
 						e.retryAfter ?? MINIMUM_BACKOFF_DELAY,
```

Comment:
I don't like this as it's a bit of a hack really and is working around the backoff api

Reply:
I changed the backoff function which leaves this bit like this. Better, worst?